tox.ini

```diff
@@ -1,3 +1,4 @@
 [flake8]
 max-line-length = 88
-extend-ignore = E501
+extend-ignore = E501, E203, E501, E266
```

Comment:
I don't think you need to list `E501` twice.

Reply:
Yeah, my oversight.